Core/Config.cpp

```diff
@@ -1,4 +1,4 @@
-// Copyright (c) 2012- PPSSPP Project.
+﻿// Copyright (c) 2012- PPSSPP Project.
```

Comment:
Ah, maybe we should comment it out for the time being, then...  Sorry.

-[Unknown]